### PR TITLE
Add option to ignore suites

### LINKED ISSUE #29: Add an option to skip talos and awsy
They are not actually suites of tests. Perhaps we should add an option in the download artifacts function to ignore suites, and by default it could be `suites_to_ignore=['talos', 'awsy']`.

### PR DESCRIPTION
**Pull request by rhcu**
Fixes #29

### CODE REVIEW

File: codecoverage.py [hunk 1]
```diff
@@ -76,9 +76,15 @@ def suite_name_from_task_name(name):
     return '-'.join(parts)
 
 
-def download_coverage_artifacts(build_task_id, suites):
+def download_coverage_artifacts(build_task_id, suites, suites_to_ignore):
```
Comment (marco-c) [inline codecoverage.py]: You can have a default for `suites_to_ignore`:
```
def download_coverage_artifacts(build_task_id, suites, suites_to_ignore=['talos', 'awsy']):
```

File: codecoverage.py [hunk 2]
```diff
@@ -104,7 +111,7 @@ def _is_chosen_task(t):
             if not any(suite in t['task']['metadata']['name'] for t in test_tasks):
                 warnings.warn('Suite %s not found' % suite)
 
-    for test_task in test_tasks:
+    for test_task in test_tasks[:1]:
```
Comment (marco-c) [inline codecoverage.py]: I assume this was for testing, it should be removed now :smiley:

File: codecoverage.py [hunk 3]
```diff
@@ -210,6 +217,7 @@ def main():
     parser.add_argument('--no-download', action='store_true', help='Use already downloaded coverage files')
     parser.add_argument('--no-grcov', action='store_true', help='Use already generated grcov output (implies --no-download)')
     parser.add_argument('--suite', action='store', nargs='+', help='List of test suites to include (by default they are all included). E.g. \'mochitest\', \'mochitest-chrome\', \'gtest\', etc.')
+    parser.add_argument('--ignore', action='store', nargs='+', help='List of test suites to ignore (by default \'tallos\' and \'awsy\'). E.g. \'mochitest\', \'mochitest-chrome\', \'gtest\', etc.')
```
Comment (marco-c) [inline codecoverage.py]: `tallos` is `talos`

File: codecoverage.py (hunk 2 above)
Comment (rhcu) [reply codecoverage.py]: Oops, yes. :)

File: codecoverage.py (hunk 1 above)
Comment (rhcu) [reply codecoverage.py]: But then ['talos', 'awsy'] will be overridden if other suites would be given as command line arguments? I thought, 'talos' and 'awsy' should always be added to ignored suites.

Comment (marco-c) [reply codecoverage.py]: No, they should be ignored by default, but the user should be able to overwrite the default.

Comment (marco-c) [reply codecoverage.py]: There might be cases where the user is interested to know the coverage given by talos or awsy.

**Pull request merged**